android/SherpaOnnxJavaDemo/sherpa/src/main/java/com/k2fsa/sherpa/onnx/AudioEvent.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 Xiaomi Corporation
```

Comment:
Could you use symlinks?

You can create a symlink to the folder of
https://github.com/k2-fsa/sherpa-onnx/tree/master/sherpa-onnx/java-api/src